src/config.zsh

```diff
@@ -30,6 +30,14 @@ ZSH_AUTOSUGGEST_ACCEPT_WIDGETS=(
 	vi-end-of-line
 )
 
+# Widgets that accept the entire suggestion and execute it
+ZSH_AUTOSUGGEST_EXECUTE_WIDGETS=(
```

Comment:
I think this array should be empty by default. All of these widgets already exist in the ACCEPT array, and shouldn't be in both. Users who want these widgets to execute instead of accept could then remove these widgets from the ACCEPT array and add to the EXECUTE array in their own configuration after sourcing the plugin